tests/fields/fields_test.go

```diff
@@ -77,3 +72,23 @@ func TestFields(t *testing.T) {
 	}
 	assert.Equal(t, want, got)
 }
+
+func TestErrorResponse(t *testing.T) {
+	ctx := context.Background()
+	plugin := loadPlugin(ctx, t)
+
+	_, err := plugin.TestError(ctx, emptypb.Empty{})
+	require.Error(t, err)
+	require.Equal(t, err.Error(), "error from plugin")
```

Comment:
can we add an additional case to ensure that empty error (errors.New("")) is also correctly handled?

Reply:
Yes, sure. Additional test added.